llmfoundry/layers_registry.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 MosaicML LLM Foundry authors
```

Comment:
just curious why is this a top level file in `llmfoundry`?

Reply:
resolved offline, may revisit in the future